collectors/logsmanagement.plugin/plugin_logsmanagement_docker_ev.c

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "plugin_logsmanagement_docker_ev.h"
+
+void docker_ev_chart_init(struct File_info *p_file_info, struct Chart_meta *chart_meta){
+    chart_meta->chart_data_docker_ev = callocz(1, sizeof (struct Chart_data_docker_ev));
+    chart_data_docker_ev_t *chart_data = chart_meta->chart_data_docker_ev;
+
+    /* Number of lines - initialise */
+    chart_data->st_lines = rrdset_create_localhost(
+            (char *) p_file_info->chart_name
+            , "lines parsed"
+            , NULL
+            , "lines parsed"
+            , NULL
+            , "Log lines parsed"
+            , "lines/s"
```

Comment:
please, remove `/s`

Reply:
only to clarify, I am requesting this, because users can change default collection, and instead to collect per second we could collect per minute, but we are always collecting line.

Reply:
I see what you are saying, but I have seen this in other places too in the agent code. Not sure what's the right approach, we need to be consistent however.

Reply:
Considering this is minor, I will mark as resolved